daml-script/daml/BUILD.bazel

```diff
@@ -63,17 +63,40 @@ filegroup(
 )
 
 genrule(
```

Comment:
Side note: We should probably create some Bazel rules for this but I’ll leave that out of this PR.